src/app/views/sidebar/resource-explorer/resource-explorer.utils.ts

```diff
@@ -126,7 +123,7 @@ export function createResourcesList(
       paths: pathItems,
       method: method?.toUpperCase(),
       type,
-      links: versionedChildren,
+      links: versionedChildren.length > 1 ? versionedChildren : [],
```

Comment:
This also disrupts the search as parent nodes with one child can't expand...
leaving it as `links: versionedChildren`  fixes the issue
```suggestion
      links: versionedChildren,
```